python/lsst/meas/algorithms/brightStarStamps.py

```diff
@@ -124,6 +125,9 @@ def measureAndNormalize(self, annulus, statsControl=afwMath.StatisticsControl(),
         # compute annularFlux
         annulusStat = afwMath.makeStatistics(annulusImage, statsFlag, statsControl)
         self.annularFlux = annulusStat.getValue()
+        if np.isnan(self.annularFlux):
+            raise ValueError("Annular flux computation failed, likely because "
```

Comment:
ValueError is "Raised when an operation or function receives an argument that has the right type but an inappropriate value,", so perhaps this should be a RuntimeError or ArithmeticError (or just make a very specific subclass called NanAnnularFluxError or similar).

Reply:
`RuntimeError` seems good to me, will update, thanks.